src/components/map/GoogleMap.svelte

```diff
@@ -51,20 +56,54 @@
 	async function loadStopsAndAddMarkers(lat, lng) {
 		const json = await loadStopsForLocation(lat, lng);
 		const stops = json.data.list;
+		allStops = [...allStops, ...stops];
```

Comment:
be sure to de-dupe these lists as you merge them.

Reply:
sure, done.